build/ci/release/Jenkinsfile

```diff
@@ -12,53 +12,41 @@ pipeline {
         VAULT_ADDR = credentials('vault-addr')
         VAULT_ROLE_ID = credentials('vault-role-id')
         VAULT_SECRET_ID = credentials('vault-secret-id')
-        REPOSITORY = "eck"
-        IMG_NAME = "eck-operator"
-        REGISTRY = "docker.elastic.co"
-        OPERATOR_IMAGE = "${REGISTRY}/${REPOSITORY}/${IMG_NAME}:${TAG_NAME}"
+
+        TAG_NAME = sh(script: 'echo -n $TAG_NAME', returnStdout: true)
```

Comment:
This is a way to try to read the variable in case it is not defined.

The `TAG_NAME` environment variable is present only when the job is built from a commit corresponding to a tag.
We use this TAG_NAME when we build a release as the version.
In the case of a nightly build, this variable does not exist. If we try to directly use the variable in this context, we got: `groovy.lang.MissingPropertyException: No such property: TAGNAME for class: groovy.lang.Binding at groovy.lang.Binding.getVariable(Binding.java:63)`.

Reply:
Ah, got it. I think this might deserve a comment.